internal/backends/mcu/lengths.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+pub struct PhysicalPx;
+pub type PhysicalLength = euclid::Length<i16, PhysicalPx>;
+pub type PhysicalRect = euclid::Rect<i16, PhysicalPx>;
+pub type PhysicalSize = euclid::Size2D<i16, PhysicalPx>;
+pub type PhysicalPoint = euclid::Point2D<i16, PhysicalPx>;
+
+pub struct LogicalPx;
+pub type LogicalLength = euclid::Length<f32, LogicalPx>;
+pub type LogicalRect = euclid::Rect<f32, LogicalPx>;
+pub type LogicalPoint = euclid::Point2D<f32, LogicalPx>;
+pub type LogicalSize = euclid::Size2D<f32, LogicalPx>;
+
+pub type ScaleFactor = euclid::Scale<f32, LogicalPx, PhysicalPx>;
+
+pub trait SizeLengths<LengthType> {
+    fn width_length(&self) -> LengthType;
```

Comment:
the euclid PR uses get_width and get_height  for this kind of stuff.
Did you depart from it on purpose?